bindings/matrix-sdk-crypto-ffi/src/machine.rs

```diff
@@ -521,6 +520,7 @@ impl OlmMachine {
         &self,
         room_id: &str,
         users: Vec<String>,
+        settings: EncryptionSettings,
```

Comment:
I assumed that rust has some kind of doc-checker to ensure all arguments are documented. Might be good to still add